cluster/photon-controller/templates/create-dynamic-salt-files.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#generate token files
+
```

Comment:
This is slightly confusing--this script (and all the scripts in templates) is not a main and is never invoked directly. Instead it is combined into a generated script in util.sh. I'll add a README explaining this so it's more clear.